ops/scripts/generate_config.sh

```diff
@@ -47,10 +47,10 @@ if ! [ -s ./.secrets ];then
     curl -s --header "PRIVATE-TOKEN: $GITLAB_PRIVATE_TOKEN" "${PROJECT_VARIABLES_URL}?per_page=100&page=1"  > .project_var_raw1
     curl -s --header "PRIVATE-TOKEN: $GITLAB_PRIVATE_TOKEN" "${PROJECT_VARIABLES_URL}?per_page=100&page=2"  > .project_var_raw2
     jq -s 'add' .project_var_raw1 .project_var_raw2 > .project_vars.json
-    cat .project_vars.json | jq --arg ENVIRONMENT $REVIEW_ENV -r '.[] | select( .environment_scope == "dev" ) | select(.key | test("key|ca|pem|cert";"i") | not ) |.key + "=" + .value' > .project_var
+    cat .project_vars.json | jq --arg ENVIRONMENT $GIGADB_ENV -r '.[] | select( .environment_scope == "*" or .environment_scope == "dev" ) | select(.key | test("key|ca|pem|cert";"i") | not ) |.key + "=" + .value' > .project_var
```

Comment:
Hi @pli888,
The FlysystemCest functional test is failng because the AWS credentials are not written to ``web.php`` config file. This is because an error on that line. 

It should be:
```
cat .project_vars.json | jq --arg ENVIRONMENT $GIGADB_ENV -r '.[] | select( .environment_scope == "dev" or  .environment_scope == "*") | select(.key | test("private_key|tlsauth|ca|pem|cert";"i") | not ) |.key + "=" + .value'
```
as currently the ``test()`` filter is filtering AWS credentials because they have ``key`` in their variable name.

Reply:
Hi @pli888,
The FlysystemCest functional test is failing on my dev also, after applying @rija's fix, the `% ./tests/unit_functional_runner` is passing now.

Reply:
`generate_config.sh` has been updated with the above line of code now.